src/lib/components/inbox/conversations/index.tsx

```diff
@@ -45,7 +45,13 @@ export class Conversations extends React.Component<Props, State> {
   }
 
   get conversations() {
-    return this.props.me.conversations.edges.map(a => a.node) || []
+    // It's currently possible for a conversation to be message-less in impulse, in which case we shouldn't show it
```

Comment:
Wondering if metaphysics should handle this - i.e. just not send a conversation at all if it doesn't contain messages. But maybe we should safeguard the client anyway?